contracts/interchain-token-service/src/contract/execute.rs

```diff
@@ -132,3 +136,25 @@ pub fn deregister_its_contract(deps: DepsMut, chain: ChainNameRaw) -> Result<Res
 
     Ok(Response::new().add_event(Event::ItsContractDeregistered { chain }.into()))
 }
+
+pub fn set_chain_config(
+    deps: DepsMut,
+    chain: ChainNameRaw,
+    max_uint: Uint256,
+    max_target_decimals: u8,
+) -> Result<Response, Error> {
+    match state::load_chain_config(deps.storage, &chain).change_context(Error::StorageError)? {
+        Some(_) => bail!(Error::ChainConfigAlreadySet(chain)),
```

Comment:
So no way to update once set?

Reply:
correct. It's permanent once set cuz if it does, it means previous transfers are all of a sudden worth more or less on the original chain or elsewhere. This is also why it resides in a separate message and state. It CANNOT be deregistered, unlike the ITS contract address.